lib/reek/sexp/sexp_formatter.rb

```diff
@@ -12,10 +12,10 @@ class SexpFormatter
       def self.format(sexp)
         return sexp.to_s unless sexp.is_a? AST::Node
         lines = Unparser.unparse(sexp).split "\n"
-        if lines.length > 1
-          "#{lines.first} ... #{lines.last}"
+        if lines.length < 3
+          lines.join('; ')
         else
-          lines.first
+          [lines.first, lines.last].join ' ... '
```

Comment:
However, I find this more confusing than  the original.

Hm .. I don't know what the cleanest way to write this whole method would be.

Perhaps it's cleanest to clearly show the three cases: one line, two lines, more lines?

```ruby
case lines.length
when 1: lines.first
when 2: lines.join('; ')
else: [lines.first, lines.last].join ' ... '
end
```